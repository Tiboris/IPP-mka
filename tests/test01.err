[['p_r', 'q'], ['0', '1'], OrderedDict(), ['p_r'], ['q']]
