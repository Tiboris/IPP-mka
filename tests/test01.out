(
{p_r, q},
{'0', '1'},
{
p_r '1' -> p_r,
p_r '0' -> q,
q '1' -> q,
q '0' -> p_r
},
p_r,
{q}
)